examples/ledFade/ledFade.ino

```diff
@@ -0,0 +1,31 @@
+/*
+  ALKS faded LED example
```

Comment:
I think that you should add information that for this example on ESP32 you need to download the [ESP32 AnalogWrite library](https://github.com/ERROPiX/ESP32_AnalogWrite).

Reply:
I am collaborating with Arduino IDE developers, that means that Arduino IDE will install all dependencies automatically in next release, this is just a draft, I am planning a release of library with this PR when there will be release of Arduino IDE.